Source/PlasticSourceControl/Private/PlasticSourceControlMenu.cpp

```diff
@@ -31,6 +33,11 @@
 #include "ToolMenus.h"
 #include "ToolMenuMisc.h"
 
+#if PLATFORM_WINDOWS
+#include "Windows/AllowWindowsPlatformTypes.h"
+#include "Windows/WindowsPlatformMisc.h"
+#endif
```

Comment:
In my local environment, these Windows<sup>TM</sup> imports make the native method `GetUserName` re-enter the game, provoking multiple compilation errors.
```suggestion
#if PLATFORM_WINDOWS
#include "Windows/AllowWindowsPlatformTypes.h"
#include "Windows/WindowsPlatformMisc.h"
#undef GetUserName
#endif
```

Reply:
Fixed, thanks!